diff --git a/sails-db/src/schema.rs b/sails-db/src/schema.rs
index 798791d..70ea096 100644
--- a/sails-db/src/schema.rs
+++ b/sails-db/src/schema.rs
@@ -26,18 +26,21 @@ table! {
         price -> BigInt,
         description -> Text,
     }
 }
 
 table! {
+    use diesel::sql_types::*;
+    use crate::enums::*;
+
     users (id) {
         id -> Text,
         name -> Text,
         school -> Text,
         hashed_passwd -> Text,
-        user_status -> Text,
+        user_status -> UserStatusMapping,
     }
 }
 
 joinable!(products -> categories (category));
 joinable!(products -> users (seller_id));
 
